plat/common/arm/time.c

```diff
@@ -149,3 +153,8 @@ void ukplat_time_init(void)
 	/* Enable timer */
 	generic_timer_enable();
 }
+
+uint32_t ukplat_time_get_irq(void)
```

Comment:
I think we should add a `ukplat_time_get_irq` implementation  in `plat/kvm/x86/time.c` too. At this moment, the build process fails for x86 images at linking, because this function is called in ` plat/kvm/irq.c`, but not defined. Something like this should work (I've tested it on my local):
```
uint32_t ukplat_time_get_irq(void)
{
	return 0;
}
```

Reply:
Done.